### PR TITLE
chore: update arch diagrams

### PR DESCRIPTION
**Pull request by chance-coleman**
## Description

Not much changed between updates, just some cleanup and clearer naming practices.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md) followed

### CODE REVIEW

**Review by noahpb — DISMISSED** [review]
lgtm!

**Review by mjnagel — CHANGES_REQUESTED** [review]
Changes requested.

File: docs/.images/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# UDS Core Diagrams
+
+## Diagram Explanations
+At this time there is three layers to our diagram that visually explain the following:
```
Comment (mjnagel) [inline docs/.images/README.md]:
```suggestion
At this time there are three layers to our diagram that visually explain the following:
```

File: docs/.images/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+1. `Overview`, basic view of what applications are present and what other services/applications they are communicating with.
+
+2. `Traffic Direction`, similar to `Overview` layer with the addition of directional arrows to represent the flow of traffic in cluster.
+
+3. `Ports and Protocols`, similar to `Traffic Direction` layer with the addition of ports and protocols.
+
+## How to Customize
+To customize the diagram, download the svg and in [draw.io](https://draw.io) create a new diagram and load in the svg.
+
+To download the [svg from github](https://github.com/defenseunicorns/uds-core/tree/main/docs/.images), select one of the svg, then select the download button in the top right of the `Preview` view.
```
Comment (mjnagel) [inline docs/.images/README.md]:
```suggestion
To download the [svg from github](https://github.com/defenseunicorns/uds-core/tree/main/docs/.images), select one of the svg files, then select the download button in the top right of the `Preview` view.
```

File: docs/.images/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Alternatively you can also download the svg from the [uds.defenseunicorns.com](https://uds.defenseunicorns.com/reference/uds-core/overview/) docsite by right clicking and selecting the `Save image as` option.
+
+It doesn't matter which svg you download, they all contain the metadata for the other layers mentioned above ( you may need to enable the layers pop up in [draw.io](https://draw.io) to change your layer ).
```
Comment (mjnagel) [inline docs/.images/README.md]:
```suggestion
It doesn't matter which svg you download, they all contain the metadata for the other layers mentioned above (you may need to enable the layers pop up in [draw.io](https://draw.io) to change your layer).
```

File: docs/.images/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+
+#### If you have suggestions for the diagrams, we welcome issues or pull requests contributes to [uds-core](https://github.com/defenseunicorns/uds-core).
```
Comment (mjnagel) [inline docs/.images/README.md]:
```suggestion
#### If you have suggestions for the diagrams, we welcome issues or pull requests contributions to [uds-core](https://github.com/defenseunicorns/uds-core).
```

**Review by mjnagel — COMMENTED** [review]
Also one q on the port-protocols diagram, we don't have `keycloak.<ADMIN_DOMAIN>` represented there, should we capture that next to/above `sso.<DOMAIN>`?

**Review by mjnagel — APPROVED** [review]

**Pull request merged**